integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/DefaultDecimalScaleMappingIT.java

```diff
@@ -37,8 +37,8 @@ public void mapping() {
 		backendMock.expectSchema( INDEX_NAME, b -> b
 				.field( "scaled", BigDecimal.class, f -> f.defaultDecimalScale( 7 ) )
 				.field( "unscaled", BigDecimal.class, f -> f.defaultDecimalScale( 0 ) )
-				.field( "defaultScaled", BigDecimal.class, f -> f.defaultDecimalScale( 2 ) )
-				.field( "hsearch", BigDecimal.class, f -> f.defaultDecimalScale( 2 ).decimalScale( 7 ) )
+				.field( "defaultScaled", BigDecimal.class, f -> { } )
+				.field( "hsearch", BigDecimal.class, f -> f.decimalScale( 7 ) )
 				.field( "notDecimal", Integer.class )
```

Comment:
Can you please remove the commit "TODOCHECK Align decimal scale ORM default" and replace it with a commit that simply adds `@Ignore( "ORM 6 does not provide a default scale in the column metadata for BigDecimal fields" )` to `DefaultDecimalScaleMappingIT`?